src/app/add-item/add-item.component.css

```diff
@@ -0,0 +1,63 @@
+    * {
+        font-size: 14px;
+        font-weight: normal;
+        color: #979797;
+    }
+    
+    mat-form-field {
+        width: 95%;
+        margin-bottom: 10px;
+    }
+    
+    mat-dialog-content {
+        overflow-y: hidden;
+    }
+    
+    .container {
+        position: relative;
+    }
+    
+    .invalid-feedback {
+        font-size: 12px;
+        color: #f44336;
+    }
+    
+    .datePicker {
+        width: 30%;
+    }
+    
+    .typedValueColor {
```

Comment:
Sometimes you are using camelCase for class names, sometimes kebab-case. Please be consistent.